lib/opacity.js

```diff
@@ -0,0 +1,10 @@
+function opacityStep(o) {
+  if (o === 0) { return 0; }
+
+  return o < 0.1 ? o.toString().split('.')[1] : o * 100;
```

Comment:
Not 100% sure I understand this logic. Don't we want to just pass along the value from the config?

Reply:
And all the elegant logic of your function finally falls into place, I'm a little slow sometimes 😊